nginx/certificates.sls

```diff
@@ -1,20 +1,20 @@
-{% from 'nginx/map.jinja' import nginx with context %}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ '/map.jinja' import nginx, pillar_namespace with context %}
 
 include:
-  - nginx.service
+  - {{ tplroot }}.service
 
-{% set certificates_path = salt['pillar.get']('nginx:certificates_path', '/etc/nginx/ssl') %}
 prepare_certificates_path_dir:
   file.directory:
-    - name: {{ certificates_path }}
+    - name: {{ nginx.certificates_path }}
     - makedirs: True
 
-{%- for dh_param, value in salt['pillar.get']('nginx:dh_param', {}).items() %}
+{%- for dh_param, value in nginx.dh_param.items() %}
```

Comment:
I think there is different failure behaviour here (Both are correct)-
`{%- for dh_param, value in salt['pillar.get']('nginx:dh_param', {}).items() %}` swallows exceptions.
`{%- for dh_param, value in nginx.dh_param.items() %}` does not swallow exception.

Reply:
True, indeed!

It works in the end thanks to the [default value](https://github.com/saltstack-formulas/nginx-formula/pull/267/commits/324b0b8a382e783f2032fab1385f803ba6d4b40d#diff-6d772ee2a0b10367d9c41fd40222747eR21) for `dh_param` added later by 324b0b8a382e783f2032fab1385f803ba6d4b40d. I can add an intermediate value and make it default to `{}` if `'dh_param' not in nginx` to be extra cautious though.

Reply:
No, it's fine. Including in defaults.yaml is 1st level of protection and that's enough! This is non-issue!